generate-stackbrew-library.sh

```diff
@@ -110,15 +110,15 @@ for version; do
 		suiteAliases=( "${suiteAliases[@]//latest-/}" )
 		variantAliases+=( "${suiteAliases[@]}" )
 
-		# calculate the intersection of parent image arches and gosu arches
+		# calculate the intersection of parent image arches
 		arches="$(jq -r --arg arches "$arches" '
 			(
 				$arches
 				| gsub("^[[:space:]]+|[[:space:]]+$"; "")
 				| split("[[:space:]]+"; "")
 			) as $parentArches
 			| .[env.version]
-			| $parentArches - ($parentArches - (.gosu.arches | keys))
+			| $parentArches
 			| join(", ")
 		' versions.json)"
```

Comment:
This whole block should be removed, and `Architectures: $arches` should be replaced with `Architectures: $(join ', ' $arches)` (effectively the reverse of what changed here in https://github.com/docker-library/redis/pull/389)

Reply:
More concretely:

```diff
diff --git a/generate-stackbrew-library.sh b/generate-stackbrew-library.sh
index 90cef22..ba4337d 100755
--- a/generate-stackbrew-library.sh
+++ b/generate-stackbrew-library.sh
@@ -110,22 +110,10 @@ for version; do
 		suiteAliases=( "${suiteAliases[@]//latest-/}" )
 		variantAliases+=( "${suiteAliases[@]}" )
 
-		# calculate the intersection of parent image arches and gosu arches
-		arches="$(jq -r --arg arches "$arches" '
-			(
-				$arches
-				| gsub("^[[:space:]]+|[[:space:]]+$"; "")
-				| split("[[:space:]]+"; "")
-			) as $parentArches
-			| .[env.version]
-			| $parentArches - ($parentArches - (.gosu.arches | keys))
-			| join(", ")
-		' versions.json)"
-
 		echo
 		cat <<-EOE
 			Tags: $(join ', ' "${variantAliases[@]}")
-			Architectures: $arches
+			Architectures: $(join ', ' $arches)
 			GitCommit: $commit
 			Directory: $dir
 		EOE
```